parm/metplus_config/plots/analyses/grid2obs/py_plotting.config_timeseries

```diff
@@ -0,0 +1,163 @@
+#!/bin/bash 
+
+# ================================================================================================
+# NCEP EMC PYTHON PLOTTING OF CAM VERIFICATION
+# 
+# CONTRIBUTORS:     Marcel Caron, marcel.caron@noaa.gov, NOAA/NWS/NCEP/EMC-VPPPGB
+#
+# PURPOSE:          Set up configurations to plot METPLUS output statistics
+#
+# DESCRIPTION:      This configuration file defines environment variables that are meant to be 
+#                   edited by the user and that are called by python scripts at runtime to define 
+#                   the plotting task, resulting in one or more graphics output files. 
+#
+# BEFORE YOU BEGIN: Make sure to set up Verif Plotting if you haven't already.  To set up, 
+#                   clone the Github repository:
+#
+#                   $ git clone https://github.com/MarcelCaron-NOAA/verif_plotting 
```

Comment:
Remove `$ git clone https://github.com/MarcelCaron-NOAA/verif_plotting `

Reply:
Done for this one and all the others in the parm directory (including the ones you didn't ask for because they were not changed for this PR, but they were in the other 3 parm files in this directory).